### ISSUE TITLE
Documentation is not clear on how to add tables

### ISSUE THREAD

**Issue by emiliemyth** [MeltanoLabs/tap-snowflake#37]
The documentation says to set "An array of the table names that you want to sync". I tried several formats, but wasn't able to get this to work. What I did get to work was to add in a `select:` in `meltano.yml`, like so:
```
    select:
    - [schema]-[table_1].*
    - [schema]-[table_2].*
    - [schema]-[table_3].*
```
This took a lot of trial and error to work out. 

It is not clear to me if setting an array of tables is the preferred method to using this select method. (Perhaps setting the tables array properly would generate this select method?)

**Comment by haleemur** [comment]
I agree the documentation could be clearer. The `tables` is meant to limit discovery to only the specified tables. This also feels a bit redundant from a user experience perspective as the information is already provided via the selection rules in `select`. Thus the user has to repeat themselves twice.
